pom.xml

```diff
@@ -57,8 +55,25 @@
         <version.arm-oraclelinux-wls-cluster-deletenode>1.0.1</version.arm-oraclelinux-wls-cluster-deletenode>
         <!--  versions  end -->
 
+        <git.tag>main</git.tag>
+        <git.repo>weblogic-azure</git.repo>
+        <git.repo.owner>oracle</git.repo.owner>
+        <artifactsLocationBase>https://raw.githubusercontent.com/${git.repo.owner}/${git.repo}/${git.tag}/weblogic-azure-vm</artifactsLocationBase>
```

Comment:
I happen to know that these properties are necessary to the proper function of the GitHub actions workflows. Say so in a comment.

Reply:
The artifactsLocationBase is used in many [mainTemplate.json](https://github.com/azure-javaee/weblogic-azure/blob/b1944d39e635a6e1c66d91b89c905f002652e1b7/weblogic-azure-vm/arm-oraclelinux-wls-cluster/addnode/src/main/arm/mainTemplate.json#L10) files, I will update the comments.